client/blocks/inline-help/inline-help-compact-results.jsx

```diff
@@ -12,7 +12,7 @@ function InlineHelpCompactResults( { helpLinks, onClick } ) {
 						onClick={ ( event ) => onClick( event, link ) }
 						tabIndex={ -1 }
 					>
-						{ preventWidows( decodeEntities( link.title ) ) }
```

Comment:
This component doesn't seem to be in use anywhere but should be covered by generic `ul` styles.